operator/v1/types_network.go

```diff
@@ -256,6 +256,40 @@ type KuryrConfig struct {
 	// size by 1.
 	// +optional
 	OpenStackServiceNetwork string `json:"openStackServiceNetwork,omitempty"`
+
+	// enableDebug set to true will enable debug logs in kuryr-kuberntes.
+	// +optional
+	EnableDebug *bool `json:"enableDebug,omitempty"`
+
+	// enablePortPrepopulation when true will make Kuryr prepopulate each newly created port
+	// pool with a minimum number of ports. Kuryr uses Neutron port pooling to fight the fact
+	// that it takes a significant amount of time to create one. Instead of creating it when
+	// pod is being deployed, Kuryr keeps a number of ports ready to be attached to pods.
+	// +optional
+	EnablePortPrepopulation *bool `json:"enablePortPrepopulation,omitempty"`
```

Comment:
again, what happens when not set and why is this a bool pointer

Reply:
Same here.